nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/test/java/org/apache/nifi/amqp/processors/ConsumeAMQPTest.java

```diff
@@ -46,6 +45,17 @@
 
 public class ConsumeAMQPTest {
 
+    @Test
+    public void testMapToStringConversion() {
```

Comment:
This test is no longer needed since its logic has been incorporated into `validateHeaderWithValueSeparatorForHeaderParameterConsumeAndTransferToSuccess`.